lib/valkyrie/persistence/active_fedora/orm/resource.rb

```diff
@@ -16,13 +16,41 @@ class Schema < ActiveTriples::Schema
     property :mime_type, predicate: ::RDF::URI("http://test.com/mime_type")
     property :original_filename, predicate: ::RDF::URI("http://test.com/original_filename")
     property :use, predicate: ::RDF::URI("http://test.com/use")
+    property :nested_resource, predicate: ::RDF::URI("http://test.com/nested_resource")
+  end
+  class NestedResource < ActiveTriples::Resource
+    def initialize(uri = RDF::Node.new, _parent = ActiveTriples::Resource.new)
+      uri = if uri.try(:node?)
```

Comment:
So it does! This basic ruby behavior is new to me.  I always thought `super` by default got the same params **this method** was invoked with, not the values from the variables the params were assigned to, after manipulation inside the method.  I guess the latter makes sense for allowing the overriding method's defaults to apply.  

I done learnt something today.  Also it makes `to_solr` seem even more gross.